src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="link-logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="ul">
+          <li class="nav-list">
+            <a
+              href="index.html"
+              class="link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav-list">
```

Comment:
```suggestion
          </li>
          
          <li class="nav-list">
```

<img width="1064" alt="image" src="https://github.com/user-attachments/assets/6d3e41ee-3799-46a9-b1f5-8e80d1b8e090">